tests/unit/modules/network/iosxr/test_iosxr_ospfv2.py

```diff
@@ -0,0 +1,308 @@
+#
+# (c) 2019, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ansible_collections.cisco.iosxr.tests.unit.compat.mock import patch
+from ansible_collections.cisco.iosxr.plugins.modules import iosxr_ospfv2
+from ansible_collections.cisco.iosxr.tests.unit.modules.utils import (
+    set_module_args,
+)
+from .iosxr_module import TestIosxrModule, load_fixture
+
+
+class TestIosxrOspfV2Module(TestIosxrModule):
+    module = iosxr_ospfv2
+
+    def setUp(self):
```

Comment:
Shouldn't we just need two mocks? https://github.com/ansible-collections/cisco.nxos/blob/master/tests/unit/modules/network/nxos/test_nxos_ospfv2.py#L39-L52

The others seem to be unnecessary.

Reply:
looks like I left commented code there,  rest is required.